src/cli/api/endpoint.go

```diff
@@ -0,0 +1,115 @@
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"code.cloudfoundry.org/cli/cf/configuration/confighelpers"
+)
+
+type APIEndpoint struct {
+	URL               string
+	SkipSSLValidation bool
+}
+
+var configFileName = func() string {
+
+	defaultCFConfigPath, _ := confighelpers.DefaultFilePath()
+	targetsPath := filepath.Join(filepath.Dir(defaultCFConfigPath), "plugins", "AutoScalerConfig")
```

Comment:
there is an existing file under ~/.cf/plugins/config.json which represent the general information for different plugins.   so, I put it into path  ~/.cf/plugins/AutoScalerConfig/config.json , instead of rename the file into autoscalerconfig.json.  
Is a longer file name better?

Reply:
if you want to put the config to a next level directory,  please use lower case and _ as the directory name